index.js

```diff
@@ -100,6 +101,7 @@ const manageRoutes = async () => {
         else if (route === "#kitreports") kitReportsScreen(auth, route);
         else if (route === "#collectionidsearch") collectionIdSearchScreen(auth, route);
         else if (route === "#reports") reportsQuery(auth, route);
+        else if (route === "#checkoutreport") checkOutReportTemplate(auth, route);
```

Comment:
Should the route be `route === "#reports"`? My understanding is that the report page will have side bar to "Shipping Report", "Check-out Report", "ABC Report", etc. Current `reportQuery` calls generate shipping reports and may be part of the proposed bigger report page.

Reply:
Reason: `#checkoutreport` prefer page to stay on the same link & to avoid landing it again on `#reports.`
TODO: Use state manager to manage routes

Reply:
Alternatively, we can put the page info in the url as a search property (for example `?page=checkoutReport#reports` or `?page=shippingReport#reports`) . So the function triggered by `#reports` hash can render accordingly based on page requirements.